Kudu.Services.Web/Content/Scripts/FileBrowser.js

```diff
@@ -170,13 +207,16 @@ $.connection.hub.start().done(function () {
 
         this.editItem = function () {
             var that = this;
-            viewModel.editText("Fetching changes...");
+            // Blank out the editor before fetching new content
+            viewModel.editText(null);
+            statusbarObj.FetchingChanges();
             viewModel.fileEdit(this);
             if(this.mime == "text/xml")
```

Comment:
sorry that didn`t notice it is not your code. 

will be good to fix it all together.